SpatialGDK/Source/Public/Schema/UnrealMetadata.h

```diff
@@ -69,4 +68,20 @@ struct UnrealMetadata : Component
 	SubobjectToOffsetMap SubobjectNameToOffset;
 };
 
+static SubobjectToOffsetMap CreateOffsetMapFromActor(AActor* Actor)
+{
+	SubobjectToOffsetMap SubobjectNameToOffset;
```

Comment:
Could we keep a cached map for each Actor (which we delete when we destroy the Actor) so we don't have to keep remaking the map?

Reply:
Wait on second thought, this information is stored in the UnrealMetadata. So we should be going there first before accessing it right?

Reply:
This is just a helper function that's used to create the UnrealMetadata::SubobjectNameToOffset. It's also used when you need the offset map when you don't have access to the metadata though.